pkgs/development/libraries/ffmpeg/generic.nix

```
@@ -71,7 +71,7 @@
 , withOpenmpt ? withFullDeps # Tracked music files decoder
 , withOpus ? withHeadlessDeps # Opus de/encoder
 , withPlacebo ? withFullDeps && !stdenv.isDarwin # libplacebo video processing library
-, withPulse ? withSmallDeps && !stdenv.isDarwin # Pulseaudio input support
+, withPulse ? withSmallDeps && !stdenv.isDarwin && !stdenv.hostPlatform.isMinGW # Pulseaudio input support
```

```suggestion
, withPulse ? withSmallDeps && stdenv.isLinux # Pulseaudio input support
```

Isn't pulse more a linux thing?

---

Pulse is mainly developed for linux but has support for others like FreeBSD and even MacOS and Windows.

---

Applied suggested change

---

I thought networking was the reason it is avaible on windows and darwin.

eg. https://www.freedesktop.org/wiki/Software/PulseAudio/Documentation/User/Network/